src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1556,6 +1557,14 @@ public String toVariableNameString(EnchantmentOffer eo) {
 				.name("Transform Reason")
 				.description("Represents a transform reason of an <a href='events.html#entity transform'>entity transform event</a>.")
 				.since("2.8.0"));
+
+    if (Skript.isRunningMinecraft(1, 20))
```

Comment:
```suggestion
		if (Skript.isRunningMinecraft(1, 20))
```
Accidentally using spaces here instead of tabs